src/Uno.Templates/content/unoapp/.template.config/TemplateWizard.json

```diff
@@ -68,7 +68,7 @@
 					"architecture": "none",
 					"authentication": "none",
 					"configuration": "false",
-					"cultures": "",
+					"cultures": "es|fr|pt-BR",
```

Comment:
Blank shouldn't specify cultures

Reply:
I don't disagree with you... however, as I recall it's hard coded that way on the CLI because we couldn't get it to work where one way we had one behavior and the other we have the other behavior. Thus it's more important to ensure that the behavior is consistent.